src/cli/subcmds/new.rs

```diff
@@ -25,11 +32,26 @@ pub fn execute(matches: &ArgMatches) -> Result<(), &'static str> {
             fs::create_dir_all(&outpath).unwrap();
         } else {
             let mut outfile = fs::File::create(&outpath).unwrap();
-            io::copy(&mut file, &mut outfile).unwrap();
+            copy(&mut file, &mut outfile).unwrap();
         }
 
     }
 
+    // Appending amethyst dependency to Cargo.toml
+    // from "folder" to "folder/" -- better safe than sorry
+    let cargo_toml_path = if project_path.ends_with("/") {
+        project_path.to_string() + "Cargo.toml"
+    } else {
+        project_path.to_string() + "/Cargo.toml"
+    };
```

Comment:
Oh, I also found something else that might help you eliminate this block. There is a method on paths called `join()` that might interest you: http://doc.rust-lang.org/std/path/struct.Path.html#method.join